uber/site_sections/registration.py

```diff
@@ -55,17 +55,6 @@ def index(self, session, message='', page='0', search_text='', uploaded_id='', o
 
         attendees = attendees.order(order)
 
-        groups = []
-        for group in (session.query(Group)
-                             .options(joinedload(Group.leader))
-                             .filter(Group.status != c.WAITLISTED,
-                                     Group.id.in_(
-                                        session.query(Attendee.group_id)
-                                               .filter(Attendee.group_id != None, Attendee.first_name == '')
-                                               .distinct()
-                                               .subquery()))):
-            groups.append((group.id, group.name + (' ({})'.format(group.leader.full_name) if group.leader else '')))
-
```

Comment:
This got moved to the ``check_in_form`` page, which is good since it knocks about 100 milliseconds off the time it takes to render this page.  Since this is probably the most commonly rendered page with dozens of check-in stations rendering it, every little bit helps :)